File: Java-OOP-Basics/Problems Course Instance 2016_2017/src/Lesson01DefiningClasses/Lab/person/Person.java
package Lesson01DefiningClasses.Lab.person;

import java.util.ArrayList;
import java.util.List;

class Person {

    private String name;
    private int age;
    private List<BankAccount> accounts;

    Person(String name, int age, List<BankAccount> accounts) {
        super();
        this.name = name;
        this.age = age;
        this.accounts = accounts;
    }

    Person(String name, int age) {
        this(name, age, new ArrayList<>());
    }

    String getName() {
        return name;
    }

    int getAge() {
        return age;
    }

    double getBalance() {
        return this.accounts.stream()
                .mapToDouble(BankAccount::getBalance)
                .sum();
    }
}

File: DB-Advanced-C#-EntityFramework-Core/Version1/DefiningClasses/DefineAClass/Person.cs
namespace DefineAClass
{
    class Person
    {
        private int age;
        private string name;

        public int Age
        {
            get
            {
                return this.age;
            }

            set
            {
                this.age = value;
            }
        }

        public string Name
        {
            get
            {
                return this.name;
            }

            set
            {
                this.name = value;
            }
        }
    }
}